### PR TITLE
Clean up/finalize deploy process

### PR DESCRIPTION
**Pull request by huang0h**
### ℹ️ Issue

Closes <issue>

### 📝 Description

- Modify update practitioner route to also re-calculate geocode to account for updated business location
- Move lots of things to env variables
- Clean up `aws_exports.js` to not expose info that'll go into env variables
- vite config update

Write a short summary of what you added. Why is it important? Any member of C4C should be able to read this and understand your contribution -- not just your team members.

Briefly list the changes made to the code:
1. Added support for this.
2. And removed redunant use of that.
3. Also this was included for reasons.

### ✔️ Verification

What steps did you take to verify your changes work? These should be clear enough for someone to be able to clone the branch and follow the steps themselves.

Provide screenshots of any new components, styling changes, or pages. 

### 🏕️ (Optional) Future Work / Notes

Did you notice anything ugly during the course of this ticket? Any bugs, design challenges, or unexpected behavior? Write it down so we can clean it up in a future ticket!

### CODE REVIEW

File: .github/workflows/ci-cd.yml [hunk 1]
```diff
@@ -121,6 +121,12 @@ jobs:
           aws_access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws_region: ${{ secrets.AWS_REGION }}
+          cognito_user_pool_id: ${{ secrets.MONARCH_COGNITO_USER_POOL_ID }}
```
Comment (huang0h) [inline .github/workflows/ci-cd.yml:124]: Moving pretty much all AWS identifiers into environment variables

File: apps/monarch/monarch-backend/src/dynamodb.ts [hunk 2]
```diff
@@ -131,10 +139,19 @@ export async function updatePractitioner(req: Request): Promise<Practitioner> {
   // Prefix with ':' to use as expression attribute values
   const practitionerData = practitionerDataFromBody(req, practitionerSchema.omit({ uuid: true }), ':');
 
+  const newGeocode = await extractGeocode(practitionerData[':businessLocation']);
```
Comment (huang0h) [inline apps/monarch/monarch-backend/src/dynamodb.ts:142]: Whenever a practitioner is updated, we should update its geocode as well to reflect a (potentially) new address

File: apps/monarch/monarch-frontend/src/app/AdminPage.tsx [hunk 3]
```diff
@@ -44,4 +44,4 @@ function AdminPage () {
     )
 }
 
-export default withAuthenticator(AdminPage);
+export default withAuthenticator(AdminPage, { hideSignUp: true });
```
Comment (huang0h) [inline apps/monarch/monarch-frontend/src/app/AdminPage.tsx:47]: This setting removes access to the "Sign up" page, only allowing users to log into existing accounts

File: apps/monarch/monarch-frontend/vite.config.ts [hunk 4]
```diff
@@ -27,9 +27,9 @@ export default defineConfig({
     }),
   ],
 
-  // define: {
-  //   global: {}
-  // },
+  define: {
```
Comment (huang0h) [inline apps/monarch/monarch-frontend/vite.config.ts:30]: Setting `global: 'globalThis` allows the app to run locally, as well as pass tests that would normally fail if it was set to `{}` as well

**Review by mattrwang — APPROVED** [review]
lgtm!

**Pull request merged**